docs/src/pages/components/data-grid/components/components.md

```diff
@@ -9,7 +9,7 @@ title: Data Grid - Components
 ## Overriding components
 
 As part of the customization API, the grid allows you to override internal components with the `components` prop.
-The prop accepts an object of type [`GridSlotsComponent`](/api/data-grid/#slots).
+The prop accepts an object of type [`GridSlotsComponent`](/api/data-grid/data-grid/#slots).
 
```

Comment:
**Off-topic**

We have talked about using these URLs in v5
```suggestion
The prop accepts an object of type [`GridSlotsComponent`](/api/x-data-grid/data-grid/#slots).
```

Using the name of the package then the name of the module exported: https://www.notion.so/mui-org/Sitemap-and-URL-structure-rearrange-ca604f628c17438db2d987961fcb8ffe